src/dmd/dsymbol.d

```diff
@@ -687,7 +687,7 @@ extern (C++) class Dsymbol : RootObject
      * Returns:
      *      symbol found, NULL if not
      */
-    final Dsymbol searchX(Loc loc, Scope* sc, RootObject id)
+    final Dsymbol searchX(Loc loc, Scope* sc, RootObject id, int flags)
```

Comment:
Please add `Params` DDoc comment.

Reply:
Normally DAutotest should check that and fail respectively (except that we broke it today - https://github.com/dlang/dmd/pull/7637)

Reply:
Is this ok?

Reply:
> Is this ok?

IMO, yes.  Though I'm looking forward to replacing `flags` with a proper `enum`.  But that's another PR.